pagopa/pn-frontend
# Pn 6315 ds bug ux mittente tooltip campo codice tassonomico

## Short description
Added icon info with tooltip to taxonomy code input in preliminary information.

## List of changes proposed in this pull request
- Added icon with tooltip
- Added new copy to notifiche.json in PA

## How to test
Login PA then create manual notification. Hover mouse to icon info in taxonomy code input. You should see a tooltip now.
